.github/workflows/check-linked-issues.yml

```diff
@@ -1,12 +1,16 @@
 name: Check linked issues
 
 on:
-  pull_request_target:
+  pull_request:
     types: [opened, edited, reopened, synchronize]
 
 jobs:
   check_pull_requests:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
```

Comment:
are you sure that issues need write access? this action is posting comments on pull requests only, as far as I remember

Reply:
I've looked through the source code and come across https://github.com/nearform/github-action-check-linked-issues/blob/496a217059004f808e81227029686db27c825378/src/action.js#L83 which needs permission to delete a comment/issue, hence the `write` permission.